cmd/algorand-indexer/daemon.go

```diff
@@ -51,6 +53,16 @@ var daemonCmd = &cobra.Command{
 
 		ctx, cf := context.WithCancel(context.Background())
 		defer cf()
+		{
+			cancelCh := make(chan os.Signal, 1)
+			signal.Notify(cancelCh, syscall.SIGTERM, syscall.SIGINT)
+			go func() {
+				<-cancelCh
+				logger.Println("Stopping Indexer.")
+				cf()
+			}()
+		}
+
```

Comment:
I don't think the scope is needed here
```suggestion
		cancelCh := make(chan os.Signal, 1)
		signal.Notify(cancelCh, syscall.SIGTERM, syscall.SIGINT)
		go func() {
			<-cancelCh
			logger.Println("Stopping Indexer.")
			cf()
		}()

```

Reply:
I wanted to hide `cancelCh`.